# Topics
map_topic_          : /sonar_occupancy_map
odometry_topic_     : /raven/odom
viewpoint_topic_    : /viewpoint_selected
viewpoint_map_topic_: /viewpoint_map_info

# Constants
ravenLength               : 1.0 #m
ravenHeight               : 1.0 #m
ravenWidth                : 1.0 #m
ravenSphereRad            : 1.0 #m
unknownThresh             : 0.90
freeThresh                : 0.05
occThresh                 : 0.01